homeassistant/components/media_player/plex.py

```diff
@@ -457,6 +479,15 @@ def force_idle(self):
         self._session = None
         self._clear_media_details()
 
+    @asyncio.coroutine
+    def async_added_to_hass(self):
+        """Callback when entity is added to hass."""
+        if PLEX_DATA not in self.hass.data:
+            self.hass.data[PLEX_DATA] = PlexData()
+        if self.machine_identifier not in \
```

Comment:
Since you want to do lookups, why not make `.devices` a dictionary mapping unique_id -> device

Reply:
In fact, does it even make sense to keep track of entity objects if you only want to keep a set of unique ids ?